tests/scripts/generate_psa_tests.py

```diff
@@ -254,13 +270,17 @@ def make_test_case(self, key: StorageKey) -> test_case.TestCase:
         if self.forward:
             extra_arguments = []
         else:
+            flags = []
             # Some test keys have the RAW_DATA type and attributes that don't
             # necessarily make sense. We do this to validate numerical
             # encodings of the attributes.
             # Raw data keys have no useful exercise anyway so there is no
             # loss of test coverage.
-            exercise = key.type.string != 'PSA_KEY_TYPE_RAW_DATA'
-            extra_arguments = ['1' if exercise else '0']
+            if key.type.string != 'PSA_KEY_TYPE_RAW_DATA':
+                flags.append('TEST_FLAG_EXERCISE')
```

Comment:
This only does what the key's usage allows. As of 6d8582d64c1de41e78e19c56f5ee191bc3e09b1d keys are generated with the usage `PSA_KEY_USAGE_EXPORT`, so only export is tested, not operations.

Reply:
But in this pull request, we don't yet know what keys can be used for which algorithm. So that's a job for https://github.com/ARMmbed/mbedtls/pull/4444.